strings.txt

```diff
@@ -16858,3 +16858,955 @@
     es = Descargando…
     pl = Pobieranie…
     nl = Downloaden...
+
+  [current_location_unknown_title]
+    comment = Заголовок ошибки при поиске поизиции которая возникает через определенное время
+    en = Continue detecting your current location?
+    ru = Продолжить поиск местоположения?
+    fr = Continuer la détection de votre emplacement actuel ?
+    da = Fortsæt med at registrere din aktuelle position?
+    id = Terus mendeteksi lokasi Anda saat ini?
+    ko = 현재 위치검색을 계속하시겠습니까?
+    sv = Fortsätt detektera din nuvarande plats?
+    tr = Geçerli konumunuzuz izlenmeye devam edilsin mi
+    uk = Продовжити пошук місцезнаходження?
+    vi = Tiếp tục phát hiện địa điểm hiện tại của bạn?
+    hu = Folytatja tartózkodási helyének keresését?
+    de = Standortsuche fortsetzen?
+    fi = Jatketaanko nykyisen paikkasi havaitsemista?
+    cs = Pokračovat se zjišťováním současné polohy?
+    it = Continuare a rilevare la posizione attuale?
+    nb = Fortsette å oppdage din gjeldende plassering?
+    zh-Hans = 继续检测您的当前位置？
+    th = ค้นหาที่ตั้งปัจจุบันต่อไปหรือไม่?
+    ja = あなたの現在地を検出し続けますか？
+    ro = Continuați cu detectarea locației dvs. curente?
+    ar = هل تريد متابعة اكتشاف موقعك الحالي؟
+    sk = Pokračovať v mazaní vašej súčasnej pozície?
+    es = ¿Continuar detectando su ubicación actual?
+    pl = Kontynuować wykrywanie bieżącej lokalizacji?
+    nl = Doorgaan met het detecteren van uw huidige locatie?
+
+  [current_location_unknown_message]
+    comment = Текст ошибки при поиске поизиции которая возникает через определенное время
+    en = Current location is unknown. Maybe you are in a building or in a tunnel.
```

Comment:
Алекс, как по английски будет лучше всего?

> On Apr 26, 2016, at 4:03 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61080729>:
> 
> > +    cs = Pokračovat se zjišťováním současné polohy?
> > +    it = Continuare a rilevare la posizione attuale?
> > +    nb = Fortsette å oppdage din gjeldende plassering?
> > +    zh-Hans = 继续检测您的当前位置？
> > +    th = ค้นหาที่ตั้งปัจจุบันต่อไปหรือไม่?
> > +    ja = あなたの現在地を検出し続けますか？
> > +    ro = Continuați cu detectarea locației dvs. curente?
> > +    ar = هل تريد متابعة اكتشاف موقعك الحالي؟
> > +    sk = Pokračovať v mazaní vašej súčasnej pozície?
> > +    es = ¿Continuar detectando su ubicación actual?
> > +    pl = Kontynuować wykrywanie bieżącej lokalizacji?
> > +    nl = Doorgaan met het detecteren van uw huidige locatie?
> > +
> > +  [current_location_unknown_message]
> > +    comment = Текст ошибки при поиске поизиции которая возникает через определенное время
> > +    en = Current location is unknown. Maybe you are in a building or in a tunnel.
> You may be
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61080729>